src/main/resources/db/migration/all/20231009162205__add_support_for_cas3_to_domain_events_table.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE domain_events ALTER COLUMN application_id DROP NOT NULL;
```

Comment:
@pezholio just an FYI of this change in case you might want to put additional requirement protections in the layer above?